lib/browser/remote/objects-registry.ts

```diff
@@ -117,18 +120,18 @@ class ObjectsRegistry {
   }
 
   // Private: Clear the storage when renderer process is destroyed.
-  registerDeleteListener (webContents, contextId) {
+  registerDeleteListener (webContents: WebContents, contextId: string) {
     // contextId => ${processHostId}-${contextCount}
     const processHostId = contextId.split('-')[0]
-    const listener = (event, deletedProcessHostId) => {
+    const listener = (event: any, deletedProcessHostId: string) => {
```

Comment:
If `event` is unused it should be `_` and `unknown` or left undefiend.
```suggestion
    const listener = (_, deletedProcessHostId: string) => {
```